src/xcode/ENA/ENA/Source/Developer Menu/Features/DMSubmissionStateViewController.swift

```diff
@@ -207,21 +207,42 @@ private extension Client {
 
 		var daysAndHours: DaysAndHours = .none
 
+
+		#if INTEROP
 		group.enter()
-		availableDays { result in
+		availableDays(forCountry: "DE") { result in
 			if case let .success(days) = result {
 				daysAndHours.days = days
 			}
 			group.leave()
 		}
 
+		group.enter()
+		availableHours(day: .formattedToday(), country: "DE") { result in
+			if case let .success(hours) = result {
+				daysAndHours.hours = hours
+			}
+			group.leave()
+		}
+		
+		#else
+
+		group.enter()
+		availableDays { result in
+			if case let .success(days) = result {
+				daysAndHours.days = days
+			}
+			group.leave()
+		}
+		
 		group.enter()
 		availableHours(day: .formattedToday()) { result in
 			if case let .success(hours) = result {
 				daysAndHours.hours = hours
 			}
 			group.leave()
 		}
+		#endif
```

Comment:
too much low-level logic for a VC (but okay since this is just the developer menu)